lb_dns.tf

```diff
@@ -0,0 +1,14 @@
+resource "exoscale_nlb" "this" {
+  zone = var.zone
+  name = format("ingress-%s", var.cluster_name)
+}
+
+resource "exoscale_domain_record" "wildcard_colorized" {
```

Comment:
I wouldn't call this "colorized", because it's just a DNS record that includes the name of the cluster, whether it's a color or not. Also it's the only wildcard record this module manages, so no need to distinguish it.